plutus-tx-plugin/test/Plugin/Strict/9.6/strictPair.pir.golden

```diff
@@ -1 +1,2 @@
-\(p : pair integer integer) -> fstPair {integer} {integer} p
\ No newline at end of file
+\(p : pair integer integer) ->
+  case integer p [(\(l : integer) (r : integer) -> l)]
```

Comment:
Well, I guess it's not entirely unreasonable that calling a builtin once can be faster than evaluating a `Case` node, two function applications and a variable lookup.

Reply:
Yeah, and cost model doesn't reflect this correctly. The cost might only even out when we compare taking both values from pair, running `FstPair` and `SndPair` both.

I don't think we can replace `FstPair` and `SndPair`